test/kuttl/multi-dcs/02-unallowedScaleDown.yaml

```diff
@@ -8,7 +8,9 @@ spec:
       - name: dc1
         rack:
           - name: rack1
+          - name: rack2
```

Comment:
I thought you could reproduce it with 2 DCs of 1 rack ?

Reply:
I wasn't able to reproduce with only 1 DC. Only with 2+ DC and 1 or more nodesPerRack

Reply:
ok :/